app/models/loader/handlers/restricted_to.rb

```diff
@@ -16,7 +16,8 @@ def store_restricted_to
         policy_restricted_to_records.each do |entry|
           id, cidr = entry
           role = ::Role[id]
-          role.restricted_to = Array(cidr)
+          # Note: Sequel upgrade requires this explicit conversion now
```

Comment:
Do we need this comment anymore? Seems like it's only really informative for the duration of this PR.

Reply:
Might be worth putting a comment here that `restricted_to` defaults to an empty array to save someone having to look it up in the DB migration...

Reply:
WIll make both changes 👍